setup.py

```diff
@@ -36,7 +36,7 @@ def __repr__(self):
 
 # Cython extension and .pyx source file names must be the same to compile
 # https://stackoverflow.com/questions/8024805/cython-compiled-c-extension-importerror-dynamic-module-does-not-define-init-fu
-modules = ['backprojection', 'radon', 'zoom', 'morphology']
+modules = ['backprojection', 'radon', 'zoom', 'morphology', 'numeric']
```

Comment:
shouldn't it be _numeric?

Reply:
No, all `.pyx` files are underscored, see L42